scripts/format-changed

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+
+    echo -n \
+"Usage: $(basename "$0")
+
+Format changed scala files. Compares against develop, so results may be unpredictable
+if your branch isn't rebased on develop / if your develop isn't up-to-date.
+"
+}
+
+function format_changed_scala_files() {
+    files=$(git diff --name-only develop..HEAD | grep '\.scala')
```

Comment:
This won't list uncommited files that are changed, which is probably the main use-case. One naive way to change this is to use something like: `git diff --name-only develop..HEAD && git diff --name-only HEAD`, or maybe use `cat`. Extra credit for using something like `uniq` to ensure duplicate arguments aren't passed in.

Reply:
I think I didn't run into this because all of my recent branches have had a bunch of intermediate commits that also touched the files with uncommitted changes. That's a good catch.

Reply:
Turns out `git diff --name-only develop` does `HEAD` + unstaged changes, so no extra credit, but this is fixed.